core/http/client/pom.xml

```diff
@@ -60,6 +60,11 @@
 			<groupId>commons-codec</groupId>
 			<artifactId>commons-codec</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rdf4j-sail-api</artifactId>
```

Comment:
You can not let the http client depend on the sail api.

Reply:
I think this is only due to the sail exception, so should be fixable if we change the exception to the rdf4jexception.